components/careers/EndSection.vue

```diff
@@ -31,8 +38,11 @@ export default {
 </script>
 
 <style lang="postcss" scoped>
-.team-image {
-  width: 100%;
+.team-image-container {
+  max-width: 100%;
   height: 100%;
 }
+.team-image-container:nth-child(2) {
+  margin-top: 100px;
+}
```

Comment:
Can we create a wrapping `div` around the two image rows and set the margin there with a class? The margin value will need to be reduced for small screens.

```
<div class="team-images">
  <v-stack class="team-image-row">...</v-stack>
  <v-stack class="team-image-row">...</v-stack>
</div>
```

```
.team-images {
  margin-top: 100px;

  @media(--viewport-small) {
    margin-top: 50px;
  }
}
```

Reply:
sure @ajithranka

Reply:
@ajithranka done. [commit](https://github.com/arcana-network/website/pull/92/commits/0c4e0055b6605a86b0c150c07fc49841b62c205f)